standard/docs/en/implementation/serialization.md

```diff
@@ -94,14 +94,18 @@ Instead, data with a one-to-many relationship can be represented using a multi-t
 
 ### Multi-table
 
-The multi-table serialization breaks key sections and building blocks of OCDS into their own tables. This supports complex data structures with one-to-many relationships, whilst allowing analysts to reconstruct the views of data they need to carry out analysis. 
+The multi-table serialization separates  objects with many to one relationships (ie. arrays) into their own tables. 
 
-Multiple tables can be packaged together as the tabs of a spreadsheet, or using the Open Knowledge data package specification. 
+Multiple tables can be packaged together as the tabs of a spreadsheet, or in a collection of CSV files.  
 
-The OCDS approach to multi-table flat serializations is currently under review. Contact the [helpdesk](../../../support/) if you wish to make use of this. 
+An example multi-table template can be found [in the sample data repository](https://github.com/open-contracting/sample-data/tree/master/flat-template).
 
-### Packaging files with meta-data
+For further information on multi-table serializations please see the [flatten tool documentation](http://flatten-tool.readthedocs.io/en/latest/).
+
+
+
+## Packaging files with meta-data
```

Comment:
Changed from 3rd level to second level section as it's not CSV specific. http://standard.open-contracting.org/1.0-dev-serialization/en/implementation/serialization/#packaging-files-with-meta-data

@timgdavies Could you check this makes sense. I did mention it in our working document, but I don't think it was clear what I meant.